tests/backup_helper.go

```diff
@@ -6933,3 +6933,34 @@ func ValidateCustomResourceRestores(ctx context1.Context, orgID string, resource
 	}
 	return nil
 }
+
+// ScaleDeploymentToDesiredReplicas scales deployments to desired replicas for migrated application namespace.
+func ScaleDeploymentToDesiredReplicas(namespace string) error {
+	var options metav1.ListOptions
+	var parsedReplicas int
+	deploymentList, err := apps.Instance().ListDeployments(namespace, options)
```

Comment:
So this will fail the test in case we are running with an app which has no deployment objects?

Reply:
Right, Hence added a check in testcase to scale deployment only if deployments exists.

Reply:
Did you try running this test with an app which does not have any deployment?

Reply:
Nop, i didn't tested since we doesn't have any spec without deployment. Ive tried this manually.

Reply:
@mkoppal-px ,Made the changes and tested with statefulset app as well.

Tested with fio.

https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/4570/console
https://aetos.pwx.purestorage.com/resultSet/testSetID/549393/testCaseID/3290565